src/hb-common.cc

```diff
@@ -671,6 +671,30 @@ parse_uint (const char **pp, const char *end, unsigned int *pv)
 }
 
 static bool
+parse_uint32 (const char **pp, const char *end, uint32_t *pv)
```

Comment:
Duplication of parse_uint, no template please as it opens another can of worms on different platforms I'd think :)